### ISSUE TITLE
Close FAHcontrol also closes FAHViewer unexpectedly in Windows

### ISSUE THREAD

**Issue by informatorius** [FoldingAtHome/fah-issues#1385]
On Windows:
1) open FAHControl
2) open FAHViewer from inside FAHControl
3) close FAHControl
=> Fahviewer closes too unexpectedly
Also happens in 7.6.10 beta

**Comment by shorttack** [comment]
@informatorius Try this: open FAHControl. Open FAHViewer (the separate app version, not within FAHControl. Close FAHControl. Is the viewer still running? I would think so.

**Comment by informatorius** [comment]
Yes then it keeps running. FahControl must not launch FahViewer as child process but separate process.
